dataeng/jobs/analytics/WarehouseTransformsCI.groovy

```diff
@@ -30,6 +30,7 @@ class WarehouseTransformsCI{
                 stringParam('JENKINS_JOB_DSL_URL', allVars.get('JENKINS_JOB_DSL_URL'), 'URL for the jenkins-job-dsl repo.')
                 stringParam('JENKINS_JOB_DSL_BRANCH', allVars.get('JENKINS_JOB_DSL_BRANCH'), 'Branch of jenkins-job-dsl repo to use.')
                 stringParam('DB_NAME', allVars.get('DB_NAME'), 'Database name used to create output schema of dbt run/tests')
+                stringParam('WITH_RETRY', allVars.get('WITH_RETRY'), 'Set this to true to retry dbt tests')
```

Comment:
Had to add this parameter with default value as 'false'

Reply:
This could also just be an environment variable. It might help avoid confusion, but who knows we may want to have this as our CI system comes under more usage.

Reply:
So in case of environment variable where should I define it ? Will it go to shell script or somewhere in DSL ?
If we can define in DSL then it is fine, defining inside shell script will be a problem because regular WT CI job uses the same shell script and we are not re-trying flaky tests  there

Reply:
@estute Sorry can you explain it a bit more

Reply:
sure thing. You can set environment variables in an `env` block, like this: https://github.com/edx/jenkins-job-dsl/blob/95ae617e7e35b4e0e76ca535afa3d96af07f4abe/devops/jobs/RunAnsible.groovy. That way, it's not a user modifiable parameter. I suppose this doesn't matter at the end of the day, because this job will not be triggered directly by users, but rather as part of an automated flow.